apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -1,19 +1,43 @@
 // @flow
 
 import * as React from 'react';
-import { View } from 'react-native';
+import { View, ScrollView } from 'react-native';
 import { StyleSheet, Text } from '@kiwicom/universal-components';
 import { withNavigation } from '@kiwicom/margarita-navigation';
+import { PassengerCard } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
 type Props = {|
   +detailId: ?string,
 |};
 
 class ResultDetail extends React.Component<Props> {
   render() {
+    const name = 'John Doe';
+    const gender = 'male';
+    const nationality = 'Russian';
+    const dateOfBirth = '22/04/1980';
+    const id = 'DF45SV9';
+    const insurance = 'Travel Inusurance Name';
+    const passengerCount = 1;
+    const bags = [
+      { count: 2, type: '40x15x30cm, 3kg' },
+      { count: 1, type: '55x20x40cm, 8kg' },
+    ];
     return (
       <View style={styles.container}>
```

Comment:
What about wrapping it to separate component. I'm worried that after we add other parts like contact details and header this file can grow quite big.

Reply:
Maybe I should be more clear. I'm thinking about this whole part which can contain more `PassengerCards` and `add` button.

<img width="196" alt="screenshot 2019-02-27 at 16 45 49" src="https://user-images.githubusercontent.com/2660330/53502937-6766f800-3aaf-11e9-9428-1282ebf670ac.png">

Reply:
Good idea 👍

Reply:
Will be implemented in next PR.